app/controllers/students_controller.rb

```diff
@@ -12,38 +12,33 @@ def new
   end
 
   def edit
+    # binding.pry
     @student = Student.find(params[:id])
   end
 
   def create
-    @student = Student.new(student_params)
-
-    respond_to do |format|
-      if @student.save
-        format.html { redirect_to @student, notice: "Student was sucessfully added." }
-        format.json { render :show, status: :created, location: @student }
-      else
-        format.html { render :new }
-        format.json { render json @student.errors, status: :unprocessable_entity }
-      end
+    @student = Student.create(student_params)
+
+    if @student.save
+      redirect_to @student, notice: "Student was sucessfully added."
+    else
+      render action: "new"
     end
   end
 
   def update
     @student = Student.find(params[:id])
     if @student.update(student_params)
-      format.html { redirect_to students_url, notice: "Student was sucessfully updated" }
+      redirect_to students_path, notice: "Student was sucessfully updated"
     else
-      format.html { render action: "edit" }
+      render action: "edit"
     end
   end
 
   def destroy
-    @student = student.find(params[:id])
+    @student = Student.find(params[:id])
```

Comment:
Looks good - next we need to add a check that the user is allowed to delete students